examples/app_unity_gain_voice/src/app_unity_gain_voice.xc

```diff
@@ -7,33 +7,28 @@
 #include <stdlib.h>  /* abs */
 #include <math.h>    /* log10 */
 
-#include "src.h"
+//#define DEBUG_PRINT_ENABLE 1 // uncomment this line to enable debug printing
```

Comment:
Rather than creating a redefinition of `DEBUG_PRINT_ENABLE` here, it'd be better to instruct people to modify the define in the Makefile to `-DDEBUG_PRINT_ENABLE=1`.